recipe/linter_hints/hints.toml

```diff
@@ -26,7 +26,7 @@ matplotlib = """\
 pathlib = """\
     As of Python 3.4+, `pathlib` is part of the standard library. \
     Recipes should drop the `pathlib` dependency as it is sunset.
-    
+
```

Comment:
Do not delete this whitespace, it's necessary for correct formatting of the message in a list.

Reply:
thanks!